src/boot/hwids/device2.json

```diff
@@ -1,4 +1,5 @@
 {
+    "type": "firmware",
```

Comment:
Please add new files for these. Could be just a copy or a real device description (i.e. AWS)

Reply:
Sure but how do you generate hwids for arbitrary smbios data? What utility was used to generate hwids for the test data?

Reply:
Basically for a data like this 
```
        {
                .manufacturer           = "First Vendor",
                .product_name           = "Device 4",
                .product_sku            = "KD03",
                .family                 = "Tablet Y",
                .baseboard_product      = "FODM2",
                .baseboard_manufacturer = "First ODM",
        },
```
I want to generate hwids for test data? Is there a documentation as to how to do that?

Reply:
The easiest way is to use real data by running `fwupdtool hwids` (or `systemd-analyse chids` from recent-enough main) on the target system.

You can just modify `src/test/test-chid.c` to generate them for arbitrary test strings.